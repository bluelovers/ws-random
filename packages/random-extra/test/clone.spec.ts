import RNGFunction from '../src/generators/function';
import RNGMathRandom2 from '../src/generators/math-random2';

describe(`RNGMathRandom2`, () =>
{

	describe(`throw error`, () => {

		test(`{}`, () =>
		{

			expect(() => {

				new RNGMathRandom2().clone({} as any);

			}).toThrowErrorMatchingSnapshot()

		});

	});

})

describe(`RNGFunction`, () =>
{

	describe(`throw error`, () => {

		test(`{}`, () =>
		{

			expect(() => {

				new RNGFunction({} as any);

			}).toThrowErrorMatchingSnapshot()

		});

	});

})
