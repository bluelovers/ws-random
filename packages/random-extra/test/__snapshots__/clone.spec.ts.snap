// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RNGFunction throw error {} 1`] = `"expected {} to be a function"`;

exports[`RNGMathRandom2 throw error {} 1`] = `"expected {} to be a function"`;
